spec/mailers/notify_mailer_spec.rb

```diff
@@ -20,11 +20,16 @@
       expect(mail.body.encoded).to match(/What is your date of birth\?:\s+27 April 1998/)
     end
 
-    it "renders upload information" do
+    it "renders requester id upload information" do
       expect(mail.body.encoded).to match("## Upload your ID")
       expect(mail.body.encoded).to match(/Proof of ID:\s+file.jpg, 24 KB/)
     end
 
+    it "renders requester address upload information" do
```

Comment:
The request used here is for self, so the uploaded info is for the subject. I don't think these test names should specify requester